services/landing-page/values-usdfdev.yaml

```diff
@@ -0,0 +1,10 @@
+landing-page:
+  pull_secret: 'pull-secret'
+  motd_url: https://raw.githubusercontent.com/lsst-dm/lsp-landing-page/master/motd/integration.md
+  host: "rubin-data-dev.slac.stanford.edu"
+  image:
+    tag: "3.0.1"
+
+pull-secret:
+  enabled: true
+  path: secret/rubin/rubin-data-dev.slac.stanford.edu/pull-secret
```

Comment:
There's no landing page service any more. This was taken over by squareone. This file therefore should be deleted.